magento/pwa-studio
# [PWA-2070]: [SEO] - Increase tap target sizes and modify text color contrasts

## Description 

Lighthouse was failing SEO/Accessibility audits related to small tap target sizes and low color contrast ratios. This PR fixes those issues.

### Changes
- Increasing Wishlist Button size to 48px by 48px (minimum size to pass Lighthouse audit).
- Adding a smaller responsive variation of AddToCart Button for mobile screens.
- Applying react-aria useButton hook on buttons for better accessibility.
- Adjusting text color contrasts for better accessibility, following  [WCAG criteria](https://www.w3.org/TR/WCAG21/#contrast-minimum) used by Lighthouse.

## Related Issue







Closes PWA-2070

## Acceptance





1. Lighthouse does not report the "Tap targets are not sized appropriately" error
2. Lighthouse does not report the "Background and foreground colors do not have a sufficient contrast ratio" error

### Verification Stakeholders



### Specification




## Verification Steps






#### Test scenario(s) for direct fix/feature





1. Run lighthouse. Verify that the above errors are not reported.
2. Verify that the "add to cart" and "wishlist" buttons still function correctly
3. Verify that the "items sorted by" message on the category and search pages is legible
4. Verify that the "unavailable for purchase" message on the product page for bundled / grouped products is legible


#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
